### DISCUSSION TITLE
Trailer/Release Announcement for my FPS made w/ enable3d

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#147] [Show and tell]
https://user-images.githubusercontent.com/43304488/113057174-b7c34f00-917a-11eb-86be-5ab077c7abc2.mov

Thanks so much to yandeu and the other contributors to enable3d for making this game possible! This is a really epic library and I loved using it for this project.

Humanoid models are from mixamo, some weapon models from clara.io, and the rest of the weapons/projectiles I modeled myself.

**Comment by N8python** [comment]
(Watch with sound for full effect)

**Reply by LuginiAndrea** [reply]
Would you consider releasing the code for others to study? It would be really helpful for my project.

**Reply by N8python** [reply]
Sure! The code is complete trash - but it is already open source! https://github.com/N8python/dualDuel

**Reply by LuginiAndrea** [reply]
can I write you in private somewhere? Have a couple of questions about it

**Reply by N8python** [reply]
Sure - what's your discord?

**Reply by LuginiAndrea** [reply]
> 
> 
> Sure - what's your discord?

this: https://discord.gg/bHUABxNy
